x-pack/plugins/serverless_search/public/application/components/connectors/elastic_managed_connector_coming_soon.tsx

```diff
@@ -0,0 +1,367 @@
+                                  />
+                                </EuiToolTip>
+                              )}
+                            </EuiFlexItem>
+                            <EuiFlexItem grow={false}>
+                              {connectorExample2 && (
+                                <EuiToolTip content={connectorExample2.name}>
+                                  <EuiIcon
+                                    size="l"
+                                    title={connectorExample2?.name}
+                                    id={connectorExample2?.serviceType}
+                                    type={connectorExample2?.iconPath || 'defaultIcon'}
+                                  />
+                                </EuiToolTip>
+                              )}
+                            </EuiFlexItem>
+                            <EuiFlexItem grow={false}>
+                              <EuiIcon color="primary" size="l" type="documents" />
+                            </EuiFlexItem>
+                            <EuiFlexItem grow={false}>
+                              {connectorExample3 && (
+                                <EuiToolTip content={connectorExample3.name}>
+                                  <EuiIcon
+                                    size="l"
+                                    title={connectorExample3?.name}
+                                    id={connectorExample3?.serviceType}
+                                    type={connectorExample3?.iconPath || 'defaultIcon'}
+                                  />
+                                </EuiToolTip>
+                              )}
+                            </EuiFlexItem>
+                            <EuiFlexItem grow={false}>
+                              {connectorExample4 && (
+                                <EuiToolTip content={connectorExample4.name}>
+                                  <EuiIcon
+                                    size="l"
+                                    title={connectorExample4?.name}
+                                    id={connectorExample4?.serviceType}
+                                    type={connectorExample4?.iconPath || 'defaultIcon'}
+                                  />
+                                </EuiToolTip>
+                              )}
+                            </EuiFlexItem>
+                          </EuiFlexGroup>
+                        </EuiFlexItem>
+                        <EuiFlexItem grow={false}>
+                          <EuiText>
+                            <p>
+                              {i18n.translate(
+                                'xpack.serverlessSearch.elasticManagedConnectorEmpty.guideOneDescription',
+                                {
+                                  defaultMessage:
+                                    "Choose from over +30 third-party data sources you'd like to sync",
```

Comment:
"Over +30" is duplicate. Either "over 30" or "30+"

Reply:
But I'm pretty sure we don't have 30+ third party sources.

Reply:
If we count for instance Jira Cloud, Jira Data center, Jira Server as 3 different connectors, and the same for Confluence and Salesforce there are 33 connectors. Otherwise there are 29. I think we might say there are "over 30" here the intention is to let know users that we offer more connectors than just the 4 third party icons we show in this screen. What do you think @leemthompo ?

![CleanShot 2024-11-05 at 11 16 36@2x](https://github.com/user-attachments/assets/696cb4c9-3c4d-4569-ac2c-dd8b18a552b0)